presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkMutableRow.java

```diff
@@ -38,6 +43,39 @@ public void setBuffer(ByteBuffer buffer)
         this.buffer = buffer;
     }
 
+    public byte[] getArray()
+    {
+        return array;
+    }
+
+    public PrestoSparkMutableRow setArray(byte[] array)
+    {
+        this.array = array;
+        return this;
+    }
+
+    public int getOffset()
+    {
+        return offset;
+    }
+
+    public PrestoSparkMutableRow setOffset(int offset)
+    {
+        this.offset = offset;
+        return this;
+    }
+
+    public int getLength()
+    {
+        return length;
```

Comment:
when the mutable row is backed by ByteBuffer, will `length` be set correctly?

Reply:
No. offset is length is set only when it is backed by the `byte[]`